plugins/email_service/app/controllers/email_service/application_controller.rb

```diff
@@ -13,21 +13,30 @@ class ApplicationController < ::DashboardController
     #     redirect_to emails_path
     #   end
     # end
- 
+
     # def restrict_access
     #   unless current_user.has_role?('cloud_support_tools_viewer')
```

Comment:
It would be good if you go over the code and remove all code which is not being used.

Reply:
removed.